modules/extension/wms/pom.xml

```diff
@@ -143,10 +143,22 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.geotools</groupId>
+      <artifactId>gt-brewer</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
     	<groupId>org.mockito</groupId>
     	<artifactId>mockito-all</artifactId>
     	<scope>test</scope>
     </dependency>
+    <dependency>
+	    <groupId>javax.mail</groupId>
+	    <artifactId>mail</artifactId>
+	    <version>1.4.7</version>
+    </dependency>
```

Comment:
@Vampouille surely this should only be a test scope dependency? Also, this change introduces new tabs. Place use spaces for indentation.